std/math.d

```diff
@@ -6220,13 +6309,25 @@ if (isFloatingPoint!(R) && isFloatingPoint!(X))
     return to;
 }
 
-// ditto
+/// ditto
 R copysign(R, X)(X to, R from) @trusted pure nothrow @nogc
 if (isIntegral!(X) && isFloatingPoint!(R))
 {
     return copysign(cast(R) to, from);
 }
 
+///
+@safe pure nothrow @nogc unittest
+{
+    assert(copysign(1.0, 1.0).feqrel(1.0) > 16);
+    assert(copysign(1.0, -0.0).feqrel(-1.0) > 16);
+    assert(copysign(1UL, -1.0).feqrel(-1.0) > 16);
```

Comment:
Maybe add `copysign(-1.0, -1.0)` too?

Reply:
`copysign` should have no imprecision. Using `feqrel` here would cause confusion by implying that it does.